rx/testing/marbles.py

```diff
@@ -91,9 +79,9 @@ def action(scheduler, state=None):
     def subscribe(observer, scheduler):
         scheduler = scheduler or new_thread_scheduler
```

Comment:
This will not work as the scheduler shadows the outer scheduler. I currently call the outer scheduler for `scheduler` and the inner subscribe scheduler for `scheduler_`, i.e `_scheduler = scheduler or scheduler_ or new_thread_scheduler`. And then use `_scheduler in your code.

Reply:
Right! I've just read your explanation in issue #285.